eucaconsole.spec

```diff
@@ -150,6 +150,7 @@ install -d $RPM_BUILD_ROOT/var/run/eucaconsole
 # Create log file
 install -d $RPM_BUILD_ROOT/var/log
 touch $RPM_BUILD_ROOT/var/log/%{name}.log
+touch $RPM_BUILD_ROOT/var/log/%{name}_startup.log
```

Comment:
This package won't build until you add that file to the %files list near the bottom.  Since it needs special permissions you will also need to use %attr to specify them in the same way as the main log file.